src/pages/packet/components/package-component.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PacketCheck from '../../../images/svg/packet-check.js'
+
+const checkColors = {
+  "build": '#E56138',
+  "engage": "#CCAD0D",
+  "recruit": "#52C2C2"
```

Comment:
Same comment as I made in the CSS file - It would be helpful to define these colors in the constants file and then reference them like `$some-color`.

Reply:
How would you do this in a JS file? Couldn't find any examples in the codebase.